tests/dummy/app/serializers/application.js

```diff
@@ -0,0 +1,3 @@
+import { Serializer } from 'ember-pouch/index';
```

Comment:
Ah, thanks. I emulated [this line](https://github.com/nolanlawson/ember-pouch/blob/master/tests/dummy/app/adapters/application.js#L1). Changed now.